BaseBundle/DependencyInjection/OpenOrchestraBaseExtension.php

```diff
@@ -32,5 +32,11 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->setParameter('open_orchestra_base.administration_languages', $config['administration_languages']);
         $container->setParameter('open_orchestra_base.encryption_key', $config['encryption_key']);
+
+        $filterStrategyFolder = $container->getParameter("kernel.root_dir")."/../".$container->getParameter('open_orchestra_filter_type_strategy_folder');
```

Comment:
the parameter should not be found like that.

You should add a configuration in the `Configuration.php` file which will be used after for the directory